example/index.js

```diff
@@ -17,35 +17,44 @@ function getReplaceStr() {
 }
 
 function getResponsive() {
-  return document.querySelector('input.responsive-input').checked;
+  return document.querySelector('input#responsive-input').checked;
 }
 
-function oneRow() {
+function ellipsize(rows) {
```

Comment:
If you're doing this, isn't it better to have only a function and calling it with the correct parameter in the `click` handler?
It'd be cleaner.

Reply:
You mean having one single click handler instead of five? I thought about that, but I actually wanted to keep the code as easy as possible to read. I can do that anyway, if you prefer.

Reply:
Ok, in this way the code is simplier to change if we'll want to add/edit features in the demo.